packages/libs/src/sdk/api/generated/full/apis/NotificationsApi.ts

```diff
@@ -84,3 +84,41 @@ export class NotificationsApi extends runtime.BaseAPI {
     }
 
 }
+
+/**
+ * @export
+ */
+export const GetNotificationsValidTypesEnum = {
```

Comment:
micro-nit: "Valid" here is probably redundant and verbose. The enum itself validates the input :-)

Reply:
well, the argument is `valid_types` which is where this name derives from. keeping for backwards compat